### ISSUE TITLE
NAT Endpoint Independent Mapping has wrong default value (True)

### ISSUE THREAD

**Issue by maxlip** [hashicorp/terraform-provider-google#10547]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.
* Please do not leave _+1_ or _me too_ comments, they generate extra noise for issue followers and do not help prioritize the request.
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
* If an issue is assigned to the `modular-magician` user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to `hashibot`, a community member has claimed the issue already.

### Terraform Version

TF Version is N/A

### Affected Resource(s)

* google_compute_router_nat
(https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_router_nat#enable_endpoint_independent_mapping)

### Terraform Configuration Files

```tf
# Problem is inappropriate default, so base documented config:
resource "google_compute_router_nat" "nat" {
  name                               = "my-router-nat"
  router                             = google_compute_router.router.name
  region                             = google_compute_router.router.region
  nat_ip_allocate_option             = "AUTO_ONLY"
  source_subnetwork_ip_ranges_to_nat = "ALL_SUBNETWORKS_ALL_IP_RANGES"
  # enable_endpoint_independent_mapping is UNDEFINED

  log_config {
    enable = true
    filter = "ERRORS_ONLY"
  }
}
```

### Debug Output

N/A

### Panic Output

N/A

### Expected Behavior

enable_endpoint_independent_mapping should be set to FALSE as per NAT/API Docs: https://cloud.google.com/nat/docs/overview#specs-rfcs

"By default, Endpoint-Independent Mapping is disabled when you create a NAT gateway."

### Actual Behavior

EIP will be enabled

### Steps to Reproduce

Just apply without specifying a value for EIP

### Important Factoids

EIP is not for everyone, and unexpected use of EIP has triggered at least one known P1 incident.

### References

* EIP was enabled in #https://github.com/GoogleCloudPlatform/magic-modules/pull/4324
* Case 29096090

**Comment by edwardmedia** [comment]
Changing it is a breaking change. Add label accordingly

**Comment by shijeesh-ns** [comment]
Looks like this got missed while updating the gcloud API.

**Comment by shijeesh-ns** [comment]
Let me try to fix this in 5.0.0 branch.

**Comment by c2thorn** [comment]
closed with https://github.com/GoogleCloudPlatform/magic-modules/pull/8600
